tests/pkg/fault-trigger/manager/vm.go

```diff
@@ -14,42 +14,45 @@
 package manager
 
 import (
-	"errors"
 	"fmt"
 	"os/exec"
 	"strings"
+	"sync"
 
 	"github.com/golang/glog"
 )
 
+type VMManager interface {
+	Name() string
+	ListVMs() ([]*VM, error)
+	StopVM(*VM) error
+	StartVM(*VM) error
+}
+
+type VirshVMManager struct {
```

Comment:
add a new file for VirshVMManager, e.g. vm_virsh.go and a new file for VirshVMManager, e.g. vm_qm.go what do you think?